app/assets/stylesheets/onboarding.scss

```diff
@@ -0,0 +1,66 @@
+@import 'variables';
+@import 'mixins';
+@import 'config/import';
+
+.onboarding-task-card {
+  background: #3b49df;
+  border-radius: $br-default;
+  padding: $su-7 $su-8 $su-8 $su-8;
+  display: none;
+
+  .task-card-title {
+    font-weight: $fw-heavy;
+    font-size: $fs-4xl;
+    line-height: $lh-tight;
+    color: #ffffff;
```

Comment:
let's use a global variable here since this is just white and we should have white available in crayons.

if i remember correctly it should be: `var(--base-inverted)`, but you can double check in `app/assets/stylesheets/config/_colors.scss`!

Reply:
I didn't use the crayon var at first because I was grepping for `#ffffff` 😆 . I didn't know that this is practically the same. Updated